openstudio-common-measures.gemspec

```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.5.0'
 
   spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.2.3'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.11'
+  spec.add_dependency 'openstudio-extension', '~> 0.3.1'
```

Comment:
👍

Reply:
These needed to be updated, but long term I'd like to not have these here, it just creates headaches, for example testing CI with multi versions of OS but getting the right gems.

Reply:
Bundle install also breaks on everything using measure gems unless they are all in sync. Maybe it is needed for developer environment, but should need to install OpenStudio standards to do bundle install of measure gems.